tests/linux_bridge_test.go

```diff
@@ -0,0 +1,104 @@
+package nmstate_tests
+
+import (
+	"io/ioutil"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	yaml "github.com/ghodss/yaml"
+
+	nmstatev1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate.io/v1"
```

Comment:
sorry for nitpicking, but can we keep internal imports as the last group (Bellow appsv1 and friends)?

Reply:
done